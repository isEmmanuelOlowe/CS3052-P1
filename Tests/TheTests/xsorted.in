100
1.413962 528.821898
1.81729961 173.987858
2.75508143 846.47578
18.6973976 860.513813
24.2991556 272.770953
25.6878246 644.591401
47.4125134 661.938885
58.8589136 691.004373
59.3686934 877.69335
73.1154782 649.664558
75.5380785 298.525606
77.0188061 841.380513
109.214906 757.048223
136.715085 943.221162
151.859864 304.657102
163.545951 899.854306
192.693937 350.221751
204.96351 591.954502
211.434985 319.529925
218.142211 479.476527
220.135508 492.109153
241.174551 44.7841404
242.884021 814.891942
250.922181 228.644317
251.315083 648.5325
253.581256 898.546969
255.986141 742.633917
261.323339 321.066976
265.612235 787.876621
286.413994 710.025581
289.029058 576.353738
302.829184 260.842552
306.785131 982.752263
343.096982 857.580462
354.533877 228.639116
372.587091 832.019301
376.551281 774.675406
377.305582 148.627986
377.647585 901.84423
383.715875 159.368627
387.476951 337.661903
390.478195 269.971118
390.61191 824.465313
393.063444 67.0302114
405.008799 761.548304
444.0966 30.6739171
446.583435 353.336097
455.090556 930.189621
472.844835 909.648273
484.633905 125.251138
498.198059 891.49522
503.910344 221.845691
516.59995 7.42826172
520.665778 424.886765
531.628252 603.616099
532.596024 462.379245
533.064714 159.071503
548.707187 270.060767
559.329326 850.811976
565.954509 718.966572
577.212317 678.15424
582.699021 604.14419
584.09022 15.5828494
608.715653 600.855136
618.194778 373.03839
633.681628 211.234979
651.269549 739.699136
652.488447 459.360316
662.83066 941.968323
669.765832 300.318248
675.540671 864.923577
680.059336 296.703348
688.256842 398.922546
695.567358 871.695461
701.624649 415.451188
742.377406 293.40057
753.553276 643.609829
761.364127 882.584573
783.411413 53.5653932
809.431695 935.560105
838.734383 693.546747
840.485369 396.464774
841.889925 611.318628
844.00846 818.766311
857.377943 404.982634
865.080021 324.021447
873.121166 222.462553
876.307513 833.147254
885.895131 212.438798
886.428433 318.692772
893.304001 398.566782
913.508169 65.6437549
916.203193 219.511285
932.180218 176.412888
935.388054 273.118281
943.395886 180.46777
943.890339 368.039867
946.370486 327.181164
959.635562 458.489973
965.732518 314.571844
