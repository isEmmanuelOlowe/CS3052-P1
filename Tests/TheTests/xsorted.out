10.9097996
